tests/amx/test_amx_instr.py

```diff
@@ -348,6 +349,12 @@ def test_matmul_on_amx_scheduled_i8(compiler, sde64):
     print("Final scheduled algorithm:")
     print(amx)
 
+    return amx
+
+def test_gen_matmul_i8_amx(matmul_i8):
```

Comment:
@gilbo Why do you need this?

Reply:
This ensures that the scheduling code will get executed on my machine even though I don't have AMX installed.  Then I can catch errors arising from the scheduling code iteslf, even if I can't run a compilation check to make sure the result is still correct.

Reply:
Ahh so it's like a wrapper around even though you don't have `compiler`